<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistDialog</class>
 <widget class="QDialog" name="RegistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regist</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="commentsLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>comments:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="commentsTextEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="privilegeLevelLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>level:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="privilegeLevelComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>manager</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>admin</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="userNameLabel">
      <property name="text">
       <string>userName:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNameLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>email:          </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="nameLable">
      <property name="text">
       <string>name:          </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>address:      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addressLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>password:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="repeatPwdLable">
      <property name="text">
       <string>repeatPwd:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="repeatPwdLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RegistDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegistDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
